cudatoolkit-dev/post-link.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash 
```

Comment:
I think we want this to be a post-install script, rather than post-link

Reply:
Oh sorry this is called `pre-link`, not post-install!

Reply:
Conda currently prints a [warning](https://github.com/conda/conda/blob/0ee04d96fd761753085dfad3f8644f0edb997b16/conda/core/link.py#L770) when a `pre-link` script is used in a package. This is why I opted for `post-link` instead. Should I change it to `pre-link` despite the warning of conda ignoring `pre-link` script in the future?

Reply:
Here's the discussion about the `pre-link` issue https://github.com/conda/conda/issues/7452

Reply:
Ahh interesting.  OK.  I didn't know about that! Thanks. This is fine then,